### PR TITLE
Add mock data for org setting page

### PR DESCRIPTION
**Pull request by SachiniSiriwardene**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: QuickStart.md [hunk 1]
```diff
@@ -248,9 +248,135 @@ After customizing the content, run “sh compress.sh {nameOfOrg}”, to create a
...
+**Update the config json**
+
+```bash
+ "identityProvider" : {
+    "name": "IS",
+    "issuer": "https://127.0.0.1:9443/oauth2/token",
+    "authorizationURL": "https://localhost:9443/oauth2/authorize",
+    "tokenURL": "https://127.0.0.1:9443/oauth2/token",
+    "userInfoURL": "https://localhost:9443/oauth2/userinfo",
+    "clientId": "meVRlVAkUoVufXXjN3aUk72PVF8a",
```
Comment (DivyaPremanantha) [inline QuickStart.md:321]: Remove client ID

File: QuickStart.md [hunk 2]
```diff
@@ -248,9 +248,135 @@ After customizing the content, run “sh compress.sh {nameOfOrg}”, to create a
...
+"superAdminRole": "superAdmin",
+````
+API request for creating IDP in developer portal
+``` bash
+curl --location --request POST 'http://localhost:3000/devportal/organizations/{organizationID}/identityProvider' \
+--header 'Content-Type: application/json' \
+--header 'Cookie: connect.sid=s%3AhKQhm7b2bCe4RkJuFknvUsxVqgG_iueA.ddy6vv265vp0cRrpRoJMnYZWs11tRTNsT0MKtTyIQ4o' \
+--data '{
+    "name": "Asgardeo",
+    "issuer": "https://api.asgardeo.io/t/sachinisiriwardene/oauth2/token",
```
Comment (DivyaPremanantha) [inline QuickStart.md:342]: let's have common names

File: QuickStart.md (hunk 1 above)
Comment (SachiniSiriwardene) [reply QuickStart.md:321]: done

File: QuickStart.md (hunk 2 above)
Comment (SachiniSiriwardene) [reply QuickStart.md]: added

**Review by DivyaPremanantha — APPROVED** [review]

**Pull request merged**